### PR TITLE
DOC: Added example of multi-line instruction in Examples section of a docstring.

### PR DESCRIPTION
**Pull request by pvanmulbregt**
Added an example showing how to use multi-line commands in the Examples section of the docstring.

### CODE REVIEW

File: doc/HOWTO_DOCUMENT.rst.txt [hunk 1]
```diff
@@ -415,6 +415,14 @@ The sections of the docstring are:
       >>> np.add([1, 2], [3, 4])
       array([4, 6])
 
+    An instruction may be split across multiple lines, with each line after
```
Comment (eric-wieser) [inline doc/HOWTO_DOCUMENT.rst.txt]: "instruction" seems like a weird description. Perhaps "An expression", "A statement", or even just "The example code"

Comment (pvanmulbregt) [reply doc/HOWTO_DOCUMENT.rst.txt]: Indeed. Thanks.

**Comment by pvanmulbregt** [comment]
Added a new PR to `numpy/numpydoc` (numpy/numpydoc#168) to put the change in `doc/format.rst`.  If that's the right place for this particular modification, then this PR can be closed without further action.

**Comment by mattip** [comment]
Thanks, yes that should do it. I will leave this open until #10878 is merged

**Comment by mattip** [comment]
numpy/numpydoc#168 was merged, thanks

**Pull request closed without merging**